docs/contributing.md

```diff
@@ -163,26 +163,24 @@ To use a preview build for a package within a project, you need to override the
 
 The [`create-release-branch`](https://github.com/MetaMask/create-release-branch) tool and [`action-publish-release`](https://github.com/MetaMask/action-publish-release) GitHub action are used to automate the release process.
 
-1. **Create a release branch.**
+1. **Initiate the Release Branch and Pre-Update Changelogs**
 
-   Run `yarn create-release-branch`. This tool generates a file and opens it in your editor, where you can specify which packages you want to include in the next release and which versions they should receive. Instructions are provided for you at the top; read them and update the file accordingly.
+   Run `yarn create-release-branch`. This command creates a release branch named `release/<new release version>` and pre-updates the changelogs for all packages with changes. It also generates a file and opens it in your editor, allowing you to specify which packages you want to include in the next release and their versions. Follow the instructions at the top of the file to update it accordingly.
```

Comment:
I'm curious the "pre-" is necessary here. It's a new change at the moment, but eventually won't be, so it seems like we don't need to bring attention to that aspect exactly. However, we do need to bring attention to the fact that changelogs have been updated with the list of changes, so that the user can use them in order to decide how to bump versions.

There's a fair amount to explain here, so perhaps we should split this into two steps?

``` suggestion
   1. **Create the release branch.**

      Start by running `yarn create-release-branch`. This command creates a branch named `release/<new release version>` which will represent the new release.

   2. **Specify packages to release along with their versions.**

      At this point, you need to tell the tool which packages you want to include in the next release and which versions to assign to those packages. You do this by modifying a YAML file called a "release spec", which the tool has generated and opened it in your editor. Follow the instructions at the top of the file to proceed.

     To assist you, the tool has also updated all of the packages that have been changed since their previous releases so that their changelogs now reflect those new changes. This should help you to understand what will be released and how to bump the versions.

     Once you save and close the release spec, the tool will proceed.
```